frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/hooks/useStrategyParameters.tsx

```diff
@@ -1,71 +1,30 @@
-import { type FC, useMemo } from 'react';
-import { StrategyEvaluationChip } from 'component/common/ConstraintsList/StrategyEvaluationChip/StrategyEvaluationChip';
-import {
-    parseParameterNumber,
-    parseParameterStrings,
-} from 'utils/parseParameter';
+import { useMemo } from 'react';
+import { parseParameterStrings } from 'utils/parseParameter';
 import { StrategyEvaluationItem } from 'component/common/ConstraintsList/StrategyEvaluationItem/StrategyEvaluationItem';
-import type { IFeatureStrategyPayload } from 'interfaces/strategy';
-import type { CreateFeatureStrategySchema } from 'openapi';
-
-const RolloutParameter: FC<{
-    value?: string | number;
-    parameters?: (
-        | IFeatureStrategyPayload
-        | CreateFeatureStrategySchema
-    )['parameters'];
-    hasConstraints?: boolean;
-    displayGroupId?: boolean;
-}> = ({ value, parameters, hasConstraints, displayGroupId }) => {
-    const percentage = parseParameterNumber(value);
-
-    const explainStickiness =
-        typeof parameters?.stickiness === 'string' &&
-        parameters?.stickiness !== 'default';
-    const stickiness = explainStickiness ? (
-        <>
-            with <strong>{parameters.stickiness}</strong>
-        </>
-    ) : (
-        ''
-    );
-
-    return (
-        <StrategyEvaluationItem type='Rollout %'>
-            <StrategyEvaluationChip label={`${percentage}%`} /> of your base{' '}
-            {stickiness}
-            <span>
-                {hasConstraints ? 'who match constraints ' : ' '}
-                is included.
-            </span>
-            {displayGroupId && parameters?.groupId ? (
-                <StrategyEvaluationChip
-                    label={`groupId: ${parameters?.groupId}`}
-                />
-            ) : null}
-        </StrategyEvaluationItem>
-    );
-};
+import type { FeatureStrategySchema } from 'openapi';
+import { RolloutParameter } from '../RolloutParameter/RolloutParameter';
 
 export const useStrategyParameters = (
-    strategy: IFeatureStrategyPayload | CreateFeatureStrategySchema,
+    strategy: Partial<FeatureStrategySchema>,
```

Comment:
What happens if the partial schema has no parameters, constraints, etc? Doesn't look like we handled it before, but now that it's a partial object, even `{}` is valid, right?

Reply:
It will return an empty array, and that is correct